en/firefox/whatsnew/whatsnew.ftl

```diff
@@ -14,10 +14,13 @@ whatsnew-up-to-date-notification-v2 = Congrats! You’re using the latest versio
 
 # Obsolete string
 whatsnew-up-to-date-notification = Congrats! You’re using the latest version of { -brand-name-firefox-browser }.
-whatsnew-out-of-date-notification-v2 = An even newer { -brand-name-firefox } is available. Restart to update.
+
+# Variables:
+#   $url (url) - link to https://support.mozilla.org/kb/update-firefox-latest-release
+whatsnew-out-of-date-notification-v3 = An even newer { -brand-name-firefox } is available. <a href="{ $url }">Update to the latest version</a>
```

Comment:
In this instance no, since the notification visually separates the link

![](https://user-images.githubusercontent.com/400117/207323482-7ddb0ecc-18cf-4a53-a3e7-227a238b0779.png)